libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -313,6 +322,7 @@ void AC_AttitudeControl::input_euler_angle_roll_pitch_yaw(float euler_roll_angle
     // Add roll trim to compensate tail rotor thrust in heli (will return zero on multirotors)
     euler_roll_angle += get_roll_trim_rad();
 
+    const float max_yaw_rate_rads = get_max_yaw_rate_rads();
```

Comment:
sorry to be nitpicky, but let's put the "max" at the end of the variable name.

Reply:
can we change this to:
get_ang_vel_yaw_max_degs

Reply:
Changed to `get_slew_ang_vel_yaw_max_rads` we already have a `get_ang_vel_yaw_max_rads`